fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -321,6 +342,23 @@ ACTOR static Future<Void> decodeBackupLogValue(Arena* arena,
 				logValue = logValue.decrypt(cipherKeys, tempArena, BlobCipherMetrics::BACKUP);
 			}
 			ASSERT(!logValue.isEncrypted());
+
+			state EncryptCipherDomainId domainId = getEncryptDomainId(logValue);
+			if (config.tenantMode == TenantMode::REQUIRED && domainId != SYSTEM_KEYSPACE_ENCRYPT_DOMAIN_ID &&
+			    domainId != FDB_DEFAULT_ENCRYPT_DOMAIN_ID) {
+				// If a tenant is not found for a given mutation then exclude it from the batch
+				ASSERT(tenantMap != nullptr);
+				if (tenantMap->find(domainId) == tenantMap->end()) {
+					// TODO (nwijetunga): Okay to log mutation here?
```

Comment:
We shouldn't log Mutation, as it might contain sensitive information? Logging TenantId and Version maybe sufficient?